README.md

```diff
@@ -1,25 +1,25 @@
 # NFT Gated Website
 
-This project demonstrates how you can restrict content on your website to only those users who own an NFT from your collection.
+This project demonstrates how you can restrict content on your website to only those users who own a token of your choice (any ERC-20, ERC-721 or ERC-1155).
 
-We use an [Edition Drop](https://portal.thirdweb.com/pre-built-contracts/edition-drop) contract to enable users to claim one of the NFTs, and serve users the restricted content if they have at least one of the NFTs claimed.
+You can use any token you like, although we will recommend you to use either [Token Drop](https://thirdweb.com/thirdweb.eth/DropERC20), [NFT Drop](https://thirdweb.com/thirdweb.eth/DropERC721) or [Edition Drop](https://thirdweb.com/thirdweb.eth/DropERC1155). That's because we will have a button to claim these test tokens for testing this template. You can also use any other token you like regardless of them being a drop token.
```

Comment:
Let's not recommend the prebuilts but let's explain if your contract is not a thirdweb one you need to go through the import flow

Reply:
Sure, although the buttons to claim the drops for testing only work with drop contracts- wouldn't it be confusing to test initially?